libdokan/fs.go

```diff
@@ -263,14 +263,22 @@ func (f *FS) open(ctx context.Context, oc *openContext, ps []string) (dokan.File
 		return nil, false, dokan.ErrObjectNameNotFound
 	case psl == 1 && ps[0] == ``:
 		return oc.returnDirNoCleanup(f.root)
+
+		// This section is equivalent to
+		// handleCommonSpecialFile in libfuse.
 	case libkbfs.ErrorFile == ps[psl-1]:
 		return NewErrorFile(f), false, nil
 	case libfs.MetricsFileName == ps[psl-1]:
 		return NewMetricsFile(f), false, nil
-	case libfs.StatusFileName == ps[0]:
-		return NewStatusFile(f.root.private.fs, nil), false, nil
+		// TODO: Port over ProfileList.
 	case libfs.ResetCachesFileName == ps[0]:
 		return &ResetCachesFile{fs: f.root.private.fs}, false, nil
+
+		// This section is equivalent to
+		// handleNonTLFSpecialFile in libfuse.
+	case libfs.StatusFileName == ps[0]:
```

Comment:
@taruti is there a reason why some cases use `ps[psl-1]` and others use `ps[0]`?

Also, `openSpecialFile` (now `handleTLFSpecialFile`) says that common files are handled in this function. But doesn't that mean that this case for `StatusFileName` (which doesn't pass in the TLF) is hit first before the case in `handleTLFSpecialFile`?